environment.yml

```diff
@@ -34,7 +34,7 @@ dependencies:
 - pip:
   - mne
   - vtk
-  - pyvista>=0.20.3
+  - https://github.com/GuillaumeFavelier/pyvista/zipball/memory_footprint
```

Comment:
I would wait for pyvista to merge the PRs so we use master here. Maybe some API we'll change still. wdyt @GuillaumeFavelier ?

Reply:
Yes, the discussion about memory optimization is still open. I would recommend waiting until it's merged then I can probably do one last pass to check that everything works as expected.